app/pages/project/vpcs/VpcPage.tsx

```diff
@@ -46,33 +46,23 @@ export default function VpcPage() {
     },
   })
 
-  const actions = useMemo(
-    () => [
-      {
-        label: 'Edit',
-        onActivate() {
-          navigate(pb.vpcEdit(vpcSelector))
-        },
-      },
-      {
-        label: 'Delete',
-        onActivate: confirmDelete({
-          doDelete: () => deleteVpc({ path: { vpc: vpcName }, query: { project } }),
-          label: vpcName,
-        }),
-        className: 'destructive',
-      },
-    ],
-    [deleteVpc, navigate, project, vpcName, vpcSelector]
-  )
-
   return (
     <>
       <PageHeader>
         <PageTitle icon={<Networking24Icon />}>{vpc.name}</PageTitle>
         <div className="inline-flex gap-2">
           <VpcDocsPopover />
-          <MoreActionsMenu label="VPC actions" actions={actions} />
+          <MoreActionsMenu label="VPC actions">
+            <DropdownMenu.LinkItem to={pb.vpcEdit(vpcSelector)}>Edit</DropdownMenu.LinkItem>
+            <DropdownMenu.Item
+              label="Delete"
+              onSelect={confirmDelete({
+                doDelete: () => deleteVpc({ path: { vpc: vpcName }, query: { project } }),
+                label: vpcName,
+              })}
+              className="destructive"
+            />
+          </MoreActionsMenu>
```

Comment:
This shows why the children approach is nice. There's no config object where you then elsewhere have to decide what it renders. You just render those things directly.